lib/slave/src/AbstractESC.cc

```diff
@@ -305,6 +328,23 @@ namespace kickcat
 
         update_process_data_input();
         update_process_data_output();
+        
+        if (not are_valid_sm(sm_mailbox_configs_))
```

Comment:
Is this check really supposed to be in op ? it's strand that it's check each time and then go to init while putting preop in error diagnostic

Reply:
Yep... it is. This test need it : 'Op > Init, SmSett0 or SmSett1 not match'

Reply:
It's seams quite brutal though (even though we did the same for the pdo sm). 
Maybe we could do something with an interruption. I'll open a issue to follow this matter

Reply:
Here's the issue